hooks/utils.py

```diff
@@ -81,6 +81,10 @@
 
 SYS_INIT_DIR = os.path.join(os.path.sep, 'etc', 'init')
 GUISERVER_INIT_PATH = os.path.join(SYS_INIT_DIR, 'guiserver.conf')
+SYSTEMD_SERVICE_DIR = os.path.join(os.path.sep, 'lib', 'systemd', 'system')
+GUISERVER_SERVICE_PATH = os.path.join(SYSTEMD_SERVICE_DIR, 'guiserver.service')
+RUNSERVER_DIR = os.path.join(os.path.sep, 'usr', 'local', 'bin')
+RUNSERVER_SH_PATH = os.path.join(RUNSERVER_DIR, 'runserver.sh')
```

Comment:
This might be a good place to comment which files are upstart and which are systemd with a brief statement about why we have to support them both, including the cut over series.  seems obvious to us all now but not so much in the future.